packages/kas/src/nodes.ts

```diff
@@ -98,25 +134,43 @@ class Expr {
     }
 
     // an abstraction for chainable, bottom-up recursion
-    recurse(method, ...passed) {
+    recurse(method: string, ...passed: any[]): this {
         var args = _.map(this.args(), function (arg) {
-            return _.isString(arg) ? arg : arg[method].apply(arg, passed);
+            return _.isString(arg) || _.isNumber(arg)
```

Comment:
Could you talk a little about the switch from `isString` to `isString || isNumber`? I see that in a couple of places. For the most part we're just changing types, but this is changing logic right?

Reply:
Sometimes the args passed to node constructors can include numbers.  This is the case for `Num` and its subclasses.  I'm not sure why the existing code hasn't caused issues previously. 🤷‍♂️